lib/internal/worker.js

```diff
@@ -490,6 +516,24 @@ function eventLoopUtilization(util1, util2) {
   );
 }
 
+let parentEventLoopStartTime = -1;
+function setParentEventLoopStartTime(time) {
+  parentEventLoopStartTime = time;
+}
+
+function parentEventLoopUtilization(util1, util2) {
+  if (parentEventLoopStartTime === -1) {
+    return { idle: 0, active: 0, utilization: 0 };
+  }
+
+  return internalEventLoopUtilization(
+    parentEventLoopStartTime,
+    parentLoopIdleTime(),
+    util1,
+    util2
```

Comment:
```suggestion
    util2,
```

Reply:
What do you mean?

Reply:
Sorry, I thought we didn't add those in function invocations. I'm ok either way.